### ISSUE TITLE
a11y: Filter search field missing label

### ISSUE THREAD

**Issue by sirrah-tam** [fusioncms/fusioncms#308]
### Describe the issue
The search filter that is used on multiple pages has a text input field that does not have a valid label. Screen reader users will not know the purpose of this input field.

### To reproduce
Open Chrome Inspector tools and locate the input field. Once highlighted, locate the Accessibility panel and its Computed Properties. The elements accessible name will appear under "Name" and it will show where the content is provided. 

### Expected behavior
All form fields need a valid label.
Reference: [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/quickref/#labels-or-instructions)

### Recommendation
The easiest way to ensure an input field has a valid label is to use the `<label>` tag with its `for` attribute referencing the ID of the input field. Developers can also use `aria-labelledby` or `aria-label` to add an accessible name.
For examples: [Labeling Controls](https://www.w3.org/WAI/tutorials/forms/labels/)

### Screenshots
<img width="1160" alt="filter field without label" src="https://user-images.githubusercontent.com/6874453/72869881-717f7f80-3c9b-11ea-8a82-51fe9da29b2e.png">

### Version and browser
Version: 6.0.0-beta.2
Browser and version: If applicable give us your browser specs

### Bug report checklist
- [X] I have filled out as much information as I can
- [X] I have detailed how the issue can be reproduced
- [X] I have [searched for existing issues](https://github.com/fusioncms/fusioncms/issues) and to the best of my knowledge this is not a duplicate

**Comment by kaidesu** [comment]
@thekanbo hold off on this one, I plan on refactoring a large portion of the datatable component in Proton that will be removing this interface.

**Comment by sirrah-tam** [comment]
@kaidesu that reminds me, I need to poke around Proton again and see if there might be any weird issues that come up like this one.
